### ISSUE TITLE
Possible exploit via .update()

### ISSUE THREAD

**Issue by githubcontrib** [knex/knex#5059]
Knex version: 1.03 and 1.04
Database + version: 10.5.9-MariaDB-1:10.5.9+maria~buster
OS: Debian GNU/Linux 10 (buster)

Hello,

I've found that the following results in an update to 'two':

```
const rogue = { one: 123, two: 456 };
const test = await knex('testing')
  .update('one', rogue)
  .where('id', 1);
```

Reproducible on a table like:

```
CREATE TABLE `testing` (
  `id` int(11) NOT NULL,
  `one` int(11) NOT NULL,
  `two` int(11) NOT NULL
);
```

Possible exploit/security issue?

Adding toString() outputs
```
update `testing` set `one` = {"one":123,"two":456} where `id` = 1
```

That's not valid SQL, but the actual query according to general_log is
```
update `testing` set `one` = `one` = 123, `two` = 456 where `id` = 1
```
Thanks

**Comment by OlivierCavadenti** [comment]
Thanks for the issue, I push a fix just now. I will check later if other statements have that kind of possible exploit.

**Comment by santialbo** [comment]
Related: https://flattsecurity.medium.com/finding-an-unseen-sql-injection-by-bypassing-escape-functions-in-mysqljs-mysql-90b27f6542b4

**Comment by kibertoad** [comment]
Released in 1.0.5
